src/CommandBasedBackendRegisterAccessor.cc

```diff
@@ -214,7 +309,21 @@ namespace ChimeraTK {
       throw ChimeraTK::runtime_error("Device not functional when reading " + this->getName());
     }
 
-    _backend->sendCommandAndRead(_writeTransferBuffer, _registerInfo.writeInfo);
+    std::vector<std::string> writeTransferBuffer =
```

Comment:
writeTransferBuffer is a bad name here, because there is also the input to the write command, which is _writeTranferBuffer.

Poposal: writeResponseBuffer